app/build.gradle

```diff
@@ -83,7 +87,14 @@ dependencies {
     compile "com.android.support:recyclerview-v7:${rootProject.ext.androidSupportSdkVersion}"
     compile "com.squareup.leakcanary:leakcanary-android:1.3.1"
 
-    apt 'com.google.dagger:dagger-compiler:2.0.1'
+    apt 'com.google.dagger:dagger-compiler:2.5'
+    apt "com.ryanharter.auto.value:auto-value-gson:0.3.1"
+    apt "com.google.auto.value:auto-value:1.2"
+    // used for AndroidStudio sync
+    provided "com.google.auto.value:auto-value:1.2"
+    // to make OkBuck work with SqlDelight
+    provided(group: 'com.github.okbuilds', name: 'SqlDelightBin', version: '1.0.0', ext: 'jar', classifier: '')
```

Comment:
Could we add dependency into the project after resolve?

If not, I think we should't add the SqlDelightBin dependency arbitrarily, it should only be added when user really use SqlDelight.